### PR TITLE
Ajmal and Antonia's weather app

### PR DESCRIPTION
**Pull request by AntoniaGranit**
https://ajmalandantoniasweatherapp.netlify.app

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+const allowLocation = () => {
+  if ("geolocation" in navigator) {
+    navigator.geolocation.getCurrentPosition(setPosition, error);
+  } else {
+    alert("Geolocation not avalible");
+  }
+};
+
+const error = () => {
```
Comment (malvinamaria) [inline script.js:19]: Nice that you added an error message when not able to use geolocation.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  fetch(styleApi)
+    .then((response) => {
+      return response.json();
+    })
+    .then((json) => {
+      const typeOfWeather = `${json.weather[0].main}`; //this creates a variable where the information
+      //about the weather condition is stored, so that it can be used in the else if-statements
+      console.log(typeOfWeather);
+      if (typeOfWeather === "Clouds") {
+        //if the json description of the weather is "clouds", the following happens
```
Comment (malvinamaria) [inline script.js:126]: Great additions with color changes depending on the weather.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+    })
+    .then((json) => {
+      updateForeCastData(json);
+    });
+
+  allowLocation();
+  cityInput.value = "";
+};
+
+const searchCity = (event) => {
```
Comment (malvinamaria) [inline script.js:195]: This is a great feature for checking the weather besides your geolocation.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  border-radius: 20px 0 0 20px;
+  box-shadow: 5px 5px #80808026;
+  outline: none;
+}
+
+.nameForm {
+  display: flex;
+}
+
+.searchButton {
```
Comment (malvinamaria) [inline style.css:107]: Beautiful looking search button.

**Review by malvinamaria — COMMENTED** [review]
The weather app looks very impressive. You have spent a lot of time on the design and it shows.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+
+.sunrise-info {
+  display: flex;
+  flex-direction: column;
+}
+
+.sunset-info {
+  display: flex;
+  flex-direction: column;
+}
```
Comment (OscarSindemark) [inline style.css:44]: It would be cool if this was centered going from mobile to desktop!

File: script.js [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+      <tr>
+      <td>${forecastDate.toLocaleString("en-US", { weekday: "long" })}</td>
+      <td class="forecasticonTd"><img src="Designs/Forecast icons/${weatherIcon}.svg" class="forecast-icon"/></td> 
+      <td class="tempTd">${value.main.temp.toFixed()} °C</td>
+      </tr>
+  `;
+  });
+};
+
+const fetchWeatherApi = (nameOfCity) => {
```
Comment (OscarSindemark) [inline script.js:90]: Really awesome that you found a way to get every single city on here, I was scared to undertake this!

**Review by OscarSindemark — APPROVED** [review]
- Your code works and is easy to follow!
- I couldn't notice any duplicates or anything
- Code followed all the blue steps. Well done!

**Pull request closed without merging**